safehttp/plugins/fetch_metadata/fetch_metadata.go

```diff
@@ -18,30 +18,28 @@ import (
 	"github.com/google/go-safeweb/safehttp"
 )
 
-const (
-	enforceMode = "enforce"
-	reportMode  = "report"
-)
-
-// LoggingService TODO
+// LoggingService is a user-provided service for logging Fetch Metadata policy
+// violations
 type LoggingService interface {
 	Log(ir *safehttp.IncomingRequest)
 }
 
 // Plugin implements Fetch Metadata functionality.
 // See https://www.w3.org/TR/fetch-metadata/ for more details.
 type Plugin struct {
-	mode   string
-	policy func(*safehttp.IncomingRequest) bool
-	logger LoggingService
+	// Policy is the Fetch Metadata policy that decides whether a request should
+	// be allowed to pass or be rejected. This will be set to a default Resource
+	// Isolation Policy by default but can be changed by the user.
+	Policy     func(*safehttp.IncomingRequest) bool
+	reportOnly bool
+	logger     LoggingService
 }
 
 // NewPlugin creates a new Fetch Metadata plugin using the defaultPolicy and
 // sets the mode to enforce.
```

Comment:
`defaultPolicy` is not visible in godoc to the users who will use the library. Explain the policy in some other way.